src/components/TodosList.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import TodoItem from "./TodoItem";
+class TodosList extends React.Component {
+  render() {
+    return (
+      <ul>
+        {this.props.todos.map(todo => (
+          <TodoItem
+          key={todo.id}
+          todo={todo}
+          handleChangeProps={this.props.handleChangeProps}
+          />
+        ))}
+      </ul>
+    )
+  }
+}
+
+export default TodosList
```

Comment:
Hello, it would be a great idea to end this line with a semi-colon. Kindly have a look at list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) for a refresher.